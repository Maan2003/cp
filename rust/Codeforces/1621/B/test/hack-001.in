1
7
4 5 4
8 15 6
8 15 2
7 14 5
4 12 5
2 5 9
9 13 6
