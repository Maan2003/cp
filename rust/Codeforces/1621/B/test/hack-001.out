4
10
6
6
6
11
11
